### PR TITLE
Add @Bromeon and @EmbersArc as CODEOWNERS for egui_plot

### PR DESCRIPTION
**Pull request by emilk**
Please take a look @Bromeon and @EmbersArc!

You have both contributed substantially to `egui_plot` (more than I have).
This is me inviting you to co-maintain the `egui_plot` crate.
If you accept, you'll get a notification when someone opens a PR touching `egui_plot`, and you can review and merge it as you see fit.
You can also merge your own egui_plot PRs without waiting on me to review them.

### CODE REVIEW

**Comment by Bromeon** [comment]
Hey, thanks a lot for the trust! 🙂 

Full disclosure, I'm not sure how much time I can commit to PR/issue reviews at the moment, as I already maintain [godot-rust](https://github.com/godot-rust/gdext) and am part of Godot and SFML teams, next to a full time job 😉 On the other hand, in 2024 I plan to work on a small game project. It might be a good chance to use egui and its plots for UIs, which would allow to dogfood some of the APIs. But I don't have a timeline yet.

TLDR: Good for me if the above expectations are OK!

**Comment by EmbersArc** [comment]
Hey, fine with me! Pretty much the same caveats apply though: Full-time job plus a game project on the side. But I'd be happy to have a look at PRs from time to time.

**Comment by emilk** [comment]
Awesome, thanks!

And yes, of course this is a "when you find time" sort of things (same as for me 😅)
